tests/integration/targets/azure_rm_cosmosdbaccount/tasks/main.yml

```
@@ -247,6 +253,54 @@
       - dbname in (output.accounts | map(attribute='name'))
       - db2name in (output.accounts | map(attribute='name'))
 
+- name: Create 4.0 Mongo free-tier instance with public networking disabled
+  azure_rm_cosmosdbaccount:
+    resource_group: "{{ resource_group }}"
+    name: "{{ dbname }}-free4"
+    location: eastasia
+    kind: mongo_db
+    mongo_version: "4.0"
+    enable_free_tier: true
```

Free tier is not supported for Internal subscriptions
```suggestion
    enable_free_tier: false
```